Lombiq.Vsix.Orchard/Services/LogWatcher/LogFileWatcherBase.cs

```diff
@@ -188,6 +193,7 @@ private static IEnumerable<string> GetAllMatchingPathsInternal(string pattern, s
                     {
                         return Directory.EnumerateFiles(combined, parts[i], SearchOption.TopDirectoryOnly);
                     }
+
                     // If this is in the middle of the path (a directory name).

```

Comment:
És az `if`-nél is így akkor.

```suggestion
                    {
                        // If this is in the middle of the path (a directory name).
```